st2common/st2common/services/action.py

```diff
@@ -216,6 +220,85 @@ def request_cancellation(liveaction, requester):
     return (liveaction, execution)
 
 
+def request_pause(liveaction, requester):
+    """
+    Request pause for a running action execution.
+
+    :return: (liveaction, execution)
+    :rtype: tuple
+    """
+    # Validate that the runner type of the action supports pause.
+    action_db = action_utils.get_action_by_ref(liveaction.action)
+
+    if not action_db:
+        raise ValueError(
+            'Unable to pause liveaction "%s" because the action "%s" '
+            'is not found.' % (liveaction.id, liveaction.action)
+        )
+
+    if action_db.runner_type['name'] not in action_constants.WORKFLOW_RUNNER_TYPES:
```

Comment:
Same here - should probably only check for Mistral until we also implement it for action-chain workflows.